src/console_reporter.cc

```diff
@@ -127,8 +127,19 @@ void ConsoleReporter::PrintRunData(const Run& result) {
         StrCat(" ", HumanReadableNumber(result.items_per_second), " items/s");
   }
 
-  const double real_time = result.GetAdjustedRealTime();
-  const double cpu_time = result.GetAdjustedCPUTime();
+  double real_time = result.GetAdjustedRealTime();
```

Comment:
Something like this seems cleaner:

```c++
auto clamp = [](double V) {
  if (V < 0 && std::fabs(V) < std::numeric_limits<double>::epsilon())
    return 0.0;
  return V;
};
const double real_time = clamp(result.GetAdjustedRealTime());
const double cpu_time = clamp(result.GetAdjustedCPUTime());
```

Reply:
*Actually, since this should probably apply to all reporters, maybe the clamping should be done in `GetAdjustedCPUTime()` et al.

What do you think?

Reply:
I think so, yes.

Actually this clamping doesn't work because `(-0 == 0) == true`, which suggests i can do something like:

```c++
auto clamp = [](double v) {
  if (std::fabs(v) < std::numeric_limits<double>::epsilon()) return 0.0;
  return v;
};
```

instead.

Reply:
Yeah, that seems to work.